### DISCUSSION TITLE
French upcoming legislation about bills' digital transmission

### DISCUSSION THREAD

**Discussion by medlouati** [PrestaShop/PrestaShop#33548] [Q&A] [answered]
Bonjour,
Je me permets de poser cette question en français compte tenu de son sujet.
Je travaille pour le compte d'un client français qui utilise la solution PrestaShop pour vendre des produits dérivés à des particuliers et entreprises. Le sujet de la réforme sur la télétransmission des factures entre entreprises (B2B) arrive en étude. En effet il a été acté que toute entreprise émettant des factures devra à terme implémenter la télétransmission, bien que la décision sur calendrier ait été repoussée à la loi de finances 2024 :
[https://www.economie.gouv.fr/cedef/facturation-electronique-entreprises](https://www.economie.gouv.fr/cedef/facturation-electronique-entreprises )
La communauté prévoit-elle le développement d'un add-on permettant l'intégration à la plateforme public ?
Infos sur la plateforme : 
[Phase pilote](https://www.economie.gouv.fr/generalisation-de-la-facturation-electronique-pour-les-entreprises-devenez-pilote-de-cette-nouvelle)
[Raccordement portail Chorus](https://portail.chorus-pro.gouv.fr/aife_csm?id=aife_csm_category&category_id=671e0b8b1b9b2410a15587b5604bcb21)

**Comment by MatShir** [comment]
It seems strongly related to this topic https://github.com/PrestaShop/PrestaShop/discussions/30844

**Reply by MatShir** [reply, marked as answer]
It is the same topic. I suggest following this discussion https://github.com/PrestaShop/PrestaShop/discussions/30844
